code/datums/antagonists/infiltrator/items/hardsuit.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/clothing/head/helmet/space/hardsuit/infiltration
+	name = "security hardsuit helmet"
```

Comment:
Not really a security hardsuit helmet, though, is it

Reply:
The name will be changed as soon as the disguise changes, tbh. But i'll change it to default